controller/cmd/tap/main.go

```diff
@@ -17,6 +17,7 @@ func main() {
 	addr := flag.String("addr", "127.0.0.1:8088", "address to serve on")
 	metricsAddr := flag.String("metrics-addr", ":9998", "address to serve scrapable metrics on")
 	kubeConfigPath := flag.String("kubeconfig", "", "path to kube config")
+	controllerNamespace := flag.String("controller-namespace", "linkerd", "namespace in which Linkerd is installed")
```

Comment:
I think this requires a change to the linkerd install template to add ` - "-controller-namespace={{.Namespace}}"` to the tap server args

Reply:
ahhhh yeah, good catch!